### DISCUSSION TITLE
Question 21 [2022]: My Parties record irn 405878  has duplicate Library [NC]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#166] [Q&A] [answered]
Description: Nina Cummings : Field Museum of Natural History - Library - Library, how would one make this so it’s just one “Library”? And not not repetitive repetitive in the Summary Line?

**Comment by magpiedin** [comment, marked as answer]
Response:  Parties irn 405878 should be fixed now

[eparties record-type workflows are here](https://docs.google.com/document/d/1RATk4HSMGNH5fRBHxRkfk6PLGJfLucmxL3DMxjSTkOM)

If other ‘Person’ Parties display duplicated department info, check the Person’s ‘Associations’ tab – If an attached preferred ‘Organization’ record includes duplicated info in its Summary string, open the Org record, go to its Organization tab and parse/re-save as needed. (You made need to edit/re-save the Person for the cleanup to also be reflected in their summary string)
